src/views/HomeView.vue

```diff
@@ -1,9 +1,158 @@
 <script setup lang="ts">
+import { ref, computed } from 'vue'
+import { useFirestore } from 'vuefire'
+import { collection, getDocs } from 'firebase/firestore'
+
+// FETCH ALL CATEGORIES AND SUBCATEGORIES
+const categories = ref<Array<{ name: string, iconURL: string, subCategories: Array<{ name: string, properties: [] }> }>>([]);
```

Comment:
Would `ref<Category[]>([])` work? Why create the class if you're not gonna use it

Reply:
Went by the user example, changed this line to include the category class

Reply:
If you check out latest master I have significantly changes the User approach, to what I think is better